src/Extension/Tasks/PhpBenchTask.yaml

```diff
@@ -0,0 +1,61 @@
+---
+id: "validation:php:benchmark"
+short_description: "Check your code efficiency by running a benchmark"
+help: ~
+stage: build
+version: 0.1.0
+command: "php %sdk_dir%/vendor/bin/phpbench run %tests% --bootstrap=%bootstrap% %iterations% %revs% %report% %time-unit%"
```

Comment:
if has default value use it like optional option

Reply:
I decided to remove default paths to not make unnecessary assumptions

Reply:
is it an optional parameter for tool or required?

Reply:
This parameters are required but can be provided via configuration file phpbench.json in project